crates/uv-workspace/src/workspace.rs

```diff
@@ -996,7 +996,7 @@ fn check_nested_workspaces(inner_workspace_root: &Path, options: &DiscoveryOptio
             if !is_excluded {
                 warn_user!(
                     "Nested workspaces are not supported, but outer workspace includes existing workspace: `{}`",
-                    pyproject_toml_path.user_display().cyan(),
+                    outer_workspace_root.simplified_display().cyan(),
```

Comment:
Running from the current directory, this message is "Nested workspaces are not supported, but outer workspace includes existing workspace: ``", seems not clear.

Reply:
Good call.